internal/nodes/avi_model_l4_translator.go

```diff
@@ -525,6 +526,28 @@ func getAutoFQDNForService(svcNamespace, svcName string) string {
 		fqdn = svcName + "." + svcNamespace + "." + subdomain
```

Comment:
@swathinsankaran : This also can lead to spill over of label if svcname or namespace is > 63.

Reply:
The kubernetes won't allow a service name and namespace beyond 63 characters in length. Hence the label won't exceed 63 characters for AutoFQDN _default_. This is the main reason why I added the changes specific to AutoFQDN _flat_.